index.js

```diff
@@ -31,6 +31,20 @@ module.exports = {
     return `<meta name="ember-cli-ifa:assetMap" content="${MetaPlaceholder}">`;
   },
 
+  /**
+   * By default, during runtime the asset-map service reads the asset map
+   * information from a meta tag on the index.html. As we do not have access to
+   * global `document` when running in fastboot, we need to implement a
+   * different way to access this asset-map information. See
+   * `get-asset-map-data` where we require the `asset-map` module taht is
+   * generated in the postBuild() below.
+   */
+  updateFastBootManifest(manifest) {
```

Comment:
FWIW, this is only ran when used as a direct project dependency (not on nested deps).